src/components/Title/style.css

```diff
@@ -13,6 +13,10 @@
   }
 }
 
+.titleSpan:focus {
+  outline: none;
```

Comment:
Why the outline here is removed? How did it end up there, because you can tab in? But then, if you can tab in, do you really want to remove outline? How the person knows where the tab is?

Reply:
This is a heading which is not normally a focusable elements. I had to add autofocus here just to make sure that page transition is announced but the element is still technically unfocusable (I followed the Google Web Fundamentals guidelines [here](https://developers.google.com/web/fundamentals/accessibility/focus/using-tabindex). Non focusable elements don't have an outline, therefore I removed it.